test/cypress/e2e/oidc/oidc_auth_test.spec.js

```diff
@@ -60,33 +58,37 @@ describe('Log in via OIDC', () => {
 
     kcLogin();
 
-    cy.origin('http://localhost:5601', () => {
-      localStorage.setItem('opendistro::security::tenant::saved', '""');
-      localStorage.setItem('home:newThemeModal:show', 'false');
+    localStorage.setItem('opendistro::security::tenant::saved', '""');
+    localStorage.setItem('home:newThemeModal:show', 'false');
 
-      cy.visit('http://localhost:5601/app/dev_tools#/console');
+    cy.visit('http://localhost:5601/app/dev_tools#/console');
 
-      cy.get('a').contains('Dev Tools').should('be.visible');
+    cy.get('a').contains('Dev Tools').should('be.visible');
 
-      cy.getCookie('security_authentication').should('exist');
-    });
+    cy.getCookie('security_authentication').should('exist');
   });
 
   it('Login to Dashboard with Hash', () => {
-    cy.visit(
-      `http://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d?_g=(filters:!(),refreshInterval:(pause:!f,value:900000),time:(from:now-24h,to:now))&_a=(description:'Analyze%20mock%20flight%20data%20for%20OpenSearch-Air,%20Logstash%20Airways,%20OpenSearch%20Dashboards%20Airlines%20and%20BeatsWest',filters:!(),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!t,title:'%5BFlights%5D%20Global%20Flight%20Dashboard',viewMode:view)`
-    );
+    const urlWithHash = `http://localhost:5601/app/security-dashboards-plugin#/getstarted`;
+
+    cy.visit(urlWithHash, {
+      failOnStatusCode: false,
+    });
 
     kcLogin();
+    cy.getCookie('security_authentication').should('exist');
+    cy.getCookie('security_authentication_oidc1').should('exist');
 
-    cy.origin('http://localhost:5601', () => {
-      localStorage.setItem('opendistro::security::tenant::saved', '""');
-      localStorage.setItem('home:newThemeModal:show', 'false');
+    cy.url().then((url) => {
```

Comment:
Is this passing the current url in the window, and use it as the origin?

Reply:
Yes, I needed to add this in so that it used the latest cookies in the browser when calling on this url. Without these lines it was not making the request with the cookies that the OIDC IdP responds with.